### DISCUSSION TITLE
Migrate Bitbucket Server and Data Center repos to GitHub.com

### DISCUSSION THREAD

**Discussion by tuves** [community/community#68208] [Enterprise]
After a successful [public beta](https://github.blog/changelog/2023-07-17-migrate-bitbucket-server-and-data-center-repos-to-github-com-public-beta/), we're launching support for Bitbucket Server and Bitbucket Data Center migrations in [GitHub Enterprise Importer](https://docs.github.com/en/migrations/using-github-enterprise-importer).

You can now easily use GitHub Enterprise Importer to migrate your source code, revision history, pull requests, reviews and comments when moving to GitHub from your self-hosted Bitbucket instance.

To learn more about the benefits of switching from Bitbucket to GitHub, check our brand new [blog post](https://github.blog/2023-09-18-switching-from-bitbucket-server-and-bamboo-to-github-just-got-easier/).

For a step-by-step guide on migrating from Bitbucket Server or Bitbucket Data Center, check out ["Migrating repositories from Bitbucket Server to GitHub Enterprise Cloud"](https://docs.github.com/en/migrations/using-github-enterprise-importer/migrating-repositories-with-github-enterprise-importer/migrating-repositories-from-bitbucket-server-to-github-enterprise-cloud) in the GitHub Docs.

**Comment by JontyRhodes14** [comment]
Migrate your repositories in batches. This will help to reduce the risk of something going wrong during the migration process.
Communicate with your team about the migration process. Let your team know when the migration will be happening and what to expect.

**Comment by xoxo2212** [comment]
ن

**Comment by JontyRhodes14** [comment]
To learn more about the benefits of switching from Bitbucket to GitHub,
